tests/test_validation.py

```diff
@@ -36,7 +36,7 @@ def test_for_valid_service_id():
 def test_all_schemas_are_valid():
     for file_name in os.listdir('json_schemas'):
         file_path = 'json_schemas/%s' % file_name
-        if os.path.isfile(file_path):
+        if os.path.isfile(file_path) and file_path.endswith(".json"):
```

Comment:
I'd be tempted to say that there probably shouldn't ever be any non-json files in the `json_schemas` directory, so throwing an error here (rather than skipping them) might be the right thing.